main.py

```diff
@@ -0,0 +1,181 @@
+    password = db.Column(db.String(60), nullable=False)
+    items = db.relationship('Item', backref='user', lazy='dynamic')
+
+    def __init__(self, username, email, image_file, password):
+        self.username = username
+        self.email = email
+        self.image_file = image_file
+        self.password = password
+
+
+class Item(db.Model):
+    __tablename__ = 'items'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(120), nullable=False)
+    description = db.Column(db.String(300), nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
+
+    def __init__(self, name, description, user_id):
+        self.name = name
+        self.description = description
+        self.user_id = user_id
+
+    def to_json(self):
+        return {
+            "name": self.name,
+            "description": self.description,
+            "user_id": self.user_id
+        }
+
+
+##### MODELS SCHEMA ######
+class userSchema(ma.Schema):
+    class Meta:
+        fields = ("username", "password")
+
+
+class itemsSchema(ma.Schema):
+    class Meta:
+        fields = ("id", "name", "description", "user_id")
+
+
+users_schema = userSchema()
+items_schema = itemsSchema()
+
+users_schema12 = userSchema(many=True)
+items_schema12 = itemsSchema(many=True)
+
+
+########## ROUTES ##########
+
+@app.route('/Registration', methods=['POST'])
+def register():
+    user_name = request.json['username']
+    email_address = request.json['email']
+    image_file = request.json['image_file']
+    passcode = request.json['password']
+    existing_user = User.query.filter_by(username=user_name).first()
+    if existing_user:
+        return jsonify(message="user already exists!"), 409
+    else:
+        new_user = User(user_name, email_address, image_file, passcode)
+        db.session.add(new_user)
+        db.session.commit()
+        return Response(status=200)
+
+
+@app.route('/Login', methods=['POST'])
+def login():
+    uname = request.json['username']
+    pswd = request.json['password']
+    existing_user = User.query.filter_by(username=uname).first()
+    if existing_user:
+        if existing_user.password == pswd:
+            return users_schema.jsonify(existing_user)
+        else:
+            return Response(status=401)
+    else:
+        return Response(status=404)
+
+
+@app.route('/addItems', methods=["POST"])
+def add_item():
+    new_item = request.json['name']
+    item_description = request.json['description']
+    user_identity = request.json['user_id']
+    new_item = Item(new_item, item_description, user_identity)
+    db.session.add(new_item)
+    db.session.commit()
+    return jsonify("New item added")
+
+
+@app.route('/remove_items/<id>', methods=['DELETE'])
+def remove_item(id):
+    item = Item.query.get(id)
+    if item:
+        db.session.delete(item)
+        db.session.commit()
+        return Response(status=200)
```

Comment:
200 is not the right status code here. Also, using this API, i can delete any user's ANY item if I know the ID. Please put the appropriate checks for authorization here.